### DISCUSSION TITLE
installation issue

### DISCUSSION THREAD

**Discussion by jh-ph** [utmstack/UTMStack#495] [Q&A] [answered]
Hi,
I'm trying to install UTMStack into an Ubuntu 22.04 LTS VM. Unfortunately, I am getting an error message referring to a vlan that the installation may have created and that it isn't defined? Just wondering if anyone have seen it or experienced it before, and know how it can be fixed. Would really want to try this out, but my lack of experience has stopped us. Appreciate any help if possible. Below is the install output:

root@phvautmstack1:/home/administrator# ./installer 

### UTMStack Installer ### 

Checking system requirements 

Checking system requirements 

Checking system requirements [OK] 

Generating Stack configuration 

Generating Stack configuration [OK] 

Configuring VLAN 

Hit:1 http://us.archive.ubuntu.com/ubuntu jammy InRelease 

Hit:2 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease 

Hit:3 http://security.ubuntu.com/ubuntu jammy-security InRelease 

Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease 

Reading package lists... Done 

Reading package lists... Done 

Building dependency tree... Done 

Reading state information... Done 

vlan is already the newest version (2.0.5ubuntu5). 

0 upgraded, 0 newly installed, 0 to remove and 174 not upgraded. 

/etc/netplan/99-vlan.yaml:9:13: Error in network definition: vlan10: interface 'ens160' is not defined 

      link: ens160 

            ^ 

exit status 78 

root@phvautmstack1:/home/administrator#  

Thanks in advance!!!
John

**Comment by c3s4rfred** [comment]
Hi, @jh-ph, check the answer of this discussion -> https://github.com/utmstack/UTMStack/discussions/437, maybe solve your issue

**Comment by jh-ph** [comment]
Hi @c3s4rfred . That discussion seems to be related to a different issue, and speaks of dockers. We are not using docker. Installing it into a VM. This is being installed on a clean OS, and tried multiple times. So, not sure what other steps we can take to get the software installed so we can at least test it.

**Reply by c3s4rfred** [reply]
Hi, @jh-ph let us know if installing from the ISO solved your problem

**Comment by jh-ph** [comment]
Hi, @c3s4rfred , we tried using the iso and were not able to get it installed. We received an error.
![image](https://github.com/utmstack/UTMStack/assets/160979607/61b62a1f-4da0-472b-bdff-d970a1d0e7dc)

Below is a crash report as well:
![image](https://github.com/utmstack/UTMStack/assets/160979607/bae46aa8-688c-44a6-9ea0-b902cae4afeb)

Not sure what our next steps should be.

Thank you.

**Comment by feynmanliang** [comment, marked as answer]
Hi, I ran into this as well after installing from Ubuntu image. When I looked in `/etc/netplan/` I noticed that the `renderer` was different between the two `.conf`s (`networkd` in the `vlan` conf and the `.conf_orig` backup, `NetworkManager` in the remaining).

After removing the `renderer: NetworkManager` conf and re-enabling the `.conf_orig` (renaming the `.conf_orig` containing `renderer: networkd` to `.conf`), I was able to get pass this VLAN error.

**Comment by jh-ph** [comment]
Hi @feynmanliang appreciate the input. That didn't work for me. I did not have the .conf files. They are .yaml files. I tried editing them and the installer just over writes it and I would get the error again. I gave up on it. Installed it in a local machine outside of my vsphere environment. The machine is running windows server 2016 and am using virtualbox to run an ubuntu OS, then followed manual installation instructions. It worked in the virtualbox vm, so I am using that to test the software out. 

Thanks again.
JH
